databricks/koalas/indexes.py

```diff
@@ -587,6 +587,67 @@ def copy(self, name=None):
             result.name = name
         return result
 
+    def droplevel(self, level):
+        """
+        Return index with requested level(s) removed.
+        If resulting index has only 1 level left, the result will be
+        of Index type, not MultiIndex.
+
+        Parameters
+        ----------
+        level : int, str, or list-like, default 0
+            If a string is given, must be the name of a level
+            If list-like, elements must be names or indexes of levels.
+
+        Returns
+        -------
+        Index or MultiIndex
+
+
+        Examples
+        --------
+        >>> midx = ks.MultiIndex.from_tuples([('a', 'x', 1), ('b', 'y', 2)],
+        ...                                  names=['level1', 'level2', 'level3'])
+        >>> midx  # doctest: +SKIP
+        MultiIndex([('a', 'x', 1),
+                    ('b', 'y', 2),
+                   names=['level1', 'level2', 'level3'])
+
+        >>> midx.droplevel([0, 1])
+        Int64Index([1, 2], dtype='int64', name='level3')
+
+        >>> midx.droplevel('level1')
+        MultiIndex([('x', 1),
+                    ('y', 2)],
+                   names=['level2', 'level3'])
+        """
+        if not isinstance(level, (tuple, list)):
+            level = [level]
+
+        for l in level:
+            if isinstance(l, int) and (l > self.nlevels - 1):
+                raise IndexError("Too many levels: Index has only {} levels, not {}"
+                                 .format(self.nlevels, l))
```

Comment:
maybe

`.format(self.nlevels, l + 1))` rather than `.format(self.nlevels, l))` ??

because pandas are doing like this.

```python
>>> pidx
MultiIndex([('a', 'x', 1),
            ('b', 'y', 2)],
           names=['level1', 'level2', 'level3'])
>>> pidx.droplevel(4)
Traceback (most recent call last):
...
packages/pandas/core/indexes/multi.py", line 1303, in _get_level_number
    "not %d" % (self.nlevels, level + 1)
IndexError: Too many levels: Index has only 3 levels, not 5
```